src/controllers/authController.js

```diff
@@ -54,3 +54,51 @@ exports.registerNewUser = (req, res) => {
 		}
 	);
 };
+
+exports.login = async (req, res) => {
+	try {
+		const prevUser = await user.findOne({ username: req.body.username });
+
+		if (prevUser) {
+			const prevUserEmail = await bcrypt.compare(
```

Comment:
is there a reason you're comparing emails? You only need to compare passwords with bcrypt.